examples/pennylane_run_barren_plateaus.py

```diff
@@ -0,0 +1,198 @@
+r"""
+.. _barren_plateaus:
+
+Barren plateaus in quantum neural networks
+==========================================
+*Author: Shahnawaz Ahmed (shahnawaz.ahmed95@gmail.com)*
+
+In classical optimization, it is suggested "that saddle
+points, not local minima, provide a fundamental impediment
+to rapid high-dimensional non-convex optimization".
+(Dauphin et. al, 2014).
```

Comment:
Are the quotation marks needed? It reads a bit oddly --- maybe move the first quotation mark, i.e., 
> it is suggested that "saddle points,
But I think it would be slightly clearer without the quotations.